Sources/StreamChatUI/Model/SendOneWallet.swift

```diff
@@ -20,14 +20,14 @@ public struct SendOneWallet {
     public var txId: String?
     public var strFormattedAmount: String?
     public var paymentTheme: String?
-    public var channelId: ChannelId?
+    public var channelId: String?
```

Comment:
@jitendra-simform  Please revert it back to ChannelId. It is easy for you to directly place channelId instead of creating everytime.

Reply:
Hello @ajay-simform  , we are sending channelId in string format as extra data , I think it's good if we pass only channel id as string
let me know if still we need to change this 
thanks